arch/arm/boot/dts/stonehenge-big-common.dtsi

```diff
@@ -73,6 +73,21 @@
 			gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
+		sd_switch {
+			label = "sd_switch";
+			gpios = <&gpio4 23 GPIO_ACTIVE_HIGH>;
+			default-state = "off";
+		};
+		gprs_switch {
```

Comment:
Ad far as I know, there's no reason for putting gprs gpio's under common stonehenge. 
About sd_switch, is ok if we want to sell "aventura" as "trail", but putting gpio's as outputs 
could increase power consumption unnecessarily